mautic/mautic
# Integration/sparkpost

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      |  No
| New feature?  |  Yes
| BC breaks?    |  None
| Deprecations? |  None
| Fixed issues  |  None

## Description
This adds the option to select the Sparkpost API transport as an email delivery option. 

## Steps to test this PR
1. Create a Sparkpost account (https://www.sparkpost.com).
2. Once your account is setup and verified, get an API key from https://app.sparkpost.com/account/credentials
3. Once you have an API key, setup your email configuration for Mautic (the url for it should look something like this http://host.yourdomain.tld/s/config/edit)
4. Add your mail delivery information (Use sandbox@sparkpostbox.com if you don't have a sending domain setup with Sparkpost).
5. Select Sparkpost as the sending service.
6. Enter your API Key where it says "Get this from your e-mail delivery service and enter it here.".
7. Click "Send test email" and it should look something like this (an api key is not shown in this picture but it has to be there for this to work).
8. Add a test subscriber and see if he gets the email.
